frontend/apps/marketing/src/selectors/contentful/getImage.ts

```diff
@@ -16,8 +17,18 @@ export function getAbsoluteImageUrl(
     const absoluteImgUrl = new URL(
       imgUrl.startsWith('//') ? `https:${imgUrl}` : imgUrl,
     );
+
     // Force AVIF format conversion for the image
     absoluteImgUrl.searchParams.set('fm', 'avif');
+
+    // Append additional parameters if provided
+    if (additionalParams) {
+      const params = new URLSearchParams(additionalParams);
+      params.forEach((value, key) => {
+        absoluteImgUrl.searchParams.set(key, value);
+      });
+    }
```

Comment:
This let's us use more parameters from the [Contentful Image API](https://www.contentful.com/developers/docs/references/images-api/).